DocSum/docker_compose/intel/hpu/gaudi/README.md

```diff
@@ -1,104 +1,120 @@
+docker build -t opea/v2a:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/dataprep/multimedia2text/video2audio/Dockerfile .
 ```
 
-### 3. Build MegaService Docker Image
+#### Multimedia2Text Service
 
-To construct the Mega Service, we utilize the [GenAIComps](https://github.com/opea-project/GenAIComps.git) microservice pipeline within the `docsum.py` Python script. Build the MegaService Docker image using the command below:
+The Multimedia2Text Service transforms multimedia data to text data. Follow these steps to build and run the service:
 
 ```bash
-git clone https://github.com/opea-project/GenAIExamples
-cd GenAIExamples/DocSum
-docker build -t opea/docsum:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f Dockerfile .
+docker build -t opea/multimedia2text:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/dataprep/multimedia2text/Dockerfile .
 ```
 
-### 4. Build UI Docker Image
+### 2. Build MegaService Docker Image
 
-Construct the frontend Docker image using the command below:
+To construct the Mega Service, we utilize the [GenAIComps](https://github.com/opea-project/GenAIComps.git) microservice pipeline within the `docsum.py` Python script. Build the MegaService Docker image via below command:
 
 ```bash
+git clone https://github.com/opea-project/GenAIExamples
 cd GenAIExamples/DocSum/
-docker build -t opea/docsum-ui:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f ./docker/Dockerfile .
+docker build -t opea/docsum:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f Dockerfile .
 ```
 
-### 5. Build React UI Docker Image
+### 4. Build UI Docker Image
 
 Build the frontend Docker image via below command:
 
 ```bash
-cd GenAIExamples/DocSum/ui
-export BACKEND_SERVICE_ENDPOINT="http://${host_ip}:8888/v1/docsum"
-docker build -t opea/docsum-react-ui:latest --build-arg BACKEND_SERVICE_ENDPOINT=$BACKEND_SERVICE_ENDPOINT -f ./docker/Dockerfile.react .
+cd GenAIExamples/DocSum/ui/gradio
+docker build -t opea/docsum-ui:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f Dockerfile .
 ```
 
-Then run the command `docker images`, you will have the following Docker Images:
-
-1. `ghcr.io/huggingface/tgi-gaudi:2.0.6`
-2. `opea/llm-docsum-tgi:latest`
-3. `opea/docsum:latest`
-4. `opea/docsum-ui:latest`
-5. `opea/docsum-react-ui:latest`
-
 ## 🚀 Start Microservices and MegaService
 
 ### Required Models
 
-We set default model as "Intel/neural-chat-7b-v3-3", change "LLM_MODEL_ID" in following setting if you want to use other models.
+We set default model as "Intel/neural-chat-7b-v3-3", change "LLM_MODEL_ID" in following Environment Variables setting if you want to use other models.
 If use gated models, you also need to provide [huggingface token](https://huggingface.co/docs/hub/security-tokens) to "HUGGINGFACEHUB_API_TOKEN" environment variable.
```

Comment:
```suggestion
Default model is "Intel/neural-chat-7b-v3-3". Change "LLM_MODEL_ID" environment variable in commands below if you want to use another model.
When using gated models, you also need to provide [HuggingFace token](https://huggingface.co/docs/hub/security-tokens) to "HUGGINGFACEHUB_API_TOKEN" environment variable.
```

(This comment applies also to all other READMEs in this PR.)

Reply:
Thank you for your suggestion. Your comment also applies to all other README files in this PR